source/stylesheets/scss/furatto/_badges.scss

```diff
@@ -0,0 +1,80 @@
+//Badges
+
+//
+//@variables
+//
+
+@import "functions";
+
+$badge-text-font-size: px-to-rems(16) !default;
+$badge-text-color: #B6B6B6 !default;
+$badge-background-color: #FF4081 !default;
+$badge-width: px-to-rems(22) !default;
+$badge-height: $badge-width !default;
+$badge-font-weight: 600 !default;
+$badge-font-size: 75% !default;
+$badge-color: #FFF !default;
+$badge-right-position: px-to-rems(-22) !default;
+$badge-top-position: px-to-rems(-11) !default;
+$badge-no-background-color: #212121 !default;
+
+
+//
+// @mixin
+//
+// We use this mixin to add badges variations
+//
+// .badge {
+//   &.primary {
+//    @include badge-background-variation($background-color: #3F51B5);
+//   }
+// }
+//
+@mixin badge-background-variation($background-color,
+                                  $color: #FFF,
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`